spec/book_spec.rb

```diff
@@ -0,0 +1,18 @@
+require 'rspec'
+require_relative '../book'
+require_relative '../rental'
+require_relative '../person'
+
+describe Book do
+  it 'Should create a new book' do
+    book = Book.new('Lord of The Rings', 'Tolkien')
+    expect(book).to be_an_instance_of(Book)
+  end
+
+  it 'Should be add as a Rental when the method "add_rental" is call' do
+    book = Book.new('Lord of The Rings', 'Tolkien')
+    person = Person.new(22, 'Austin')
+    result = book.add_rental(person, '2010/07/01')
+    expect(result).to be_an_instance_of(Rental)
+  end
+end
```

Comment:
- You have done great so far :+1: :+1: But there are a few requirements you still need to meet before moving to the next milestone. :+1: :100: 

- - 
![Selection_016](https://github.com/Xghayor/oop-school-library/assets/107632780/6e00082a-ec9a-49db-8252-8e6c5cf2787e)

- [ ] As stated in the project requirement, you need to also write tests for all the `Decorator` classes and methods. Currently, the tests for these have not been implemented. I suggest you implement them. :+1: :smile: